schema/workbook.json

```diff
@@ -1,2 +1,754 @@
+                            "items": [
+                                {
+                                    "$ref": "#/definitions/metricFilters"
+                                }
+                            ]
+                        },
+                        "gridFormatType": {
+                            "type": "integer",
+                            "description": "The format in which the metric data should be displayed in the grid"
+                        },
+                        "resourceLimit": {
+                            "type": "integer",
+                            "description": "The maximum number of resources that can be queried"
+                        },
+                        "gridSettings": {
+                            "$ref": "#/definitions/gridSettings"
+                        },
+                        "exportToExcelOptions": {
+                            "$ref": "#/definitions/exportToExcelOptions"
+                        },
+                        "showExportToExcel": {
+                            "$ref": "#/definitions/showExportToExcel"
+                        },
+                        "title": {
+                            "$ref": "#/definitions/chartTitle"
+                        }
+                    }
+                },
+                "name": {
+                    "$ref": "#/definitions/name"
+                },
+                "customWidth": {
+                    "$ref": "#/definitions/customWidth"
+                },
+                "showPin": {
+                    "$ref": "#/definitions/showPin"
+                },
+                "styleSettings": {
+                    "$ref": "#/definitions/styleSettings"
+                },
+                "conditionalVisibility": {
+                    "$ref": "#/definitions/conditionalVisibility"
+                }
+            }
+        },
+        "link": {
+            "$id": "#/definitions/link",
+            "type": "object",
+            "required": [
+                "type",
+                "content"
+            ],
+            "properties": {
+                "type": {
+                    "const": 11
+                },
+                "content": {
+                    "type": "object",
+                    "properties": {
+                        "version": {
+                            "type": "string",
+                            "description": "Version of link step",
+                            "examples": [
+                                "LinkItem/2.0"
+                            ]
+                        },
+                        "style": {
+                            "type": "string",
+                            "description": "style of link",
+                            "examples": [
+                                "bullets",
+                                "paragraph",
+                                "navigation"
```

Comment:
`tabs` is missing from the values here, and it's a big one.  technically navigation is actually `nav`:
```
const enum ViewStyle {
    Bullets = "bullets",
    List = "list",
    Paragraph = "paragraph",
    Nav = "nav",
    Tabs = "tabs",
}
```

is there a thing for the actual allowed values when they are enums?